### DISCUSSION TITLE
change infinisport port

### DISCUSSION THREAD

**Discussion by iboumedien** [infinispan/infinispan#12633] [Q&A] [answered]
Hi guys,

I'm faing an issue, when I try to change the REST port in the infinispan.xml, nothing happen, and I still have 11222. I'm using the version is 15.0.5.Final.
From : 
<socket-binding name="default" port="${infinispan.bind.port:11222}"/
To : 
<socket-binding name="default" port="${infinispan.bind.port:11232}"/

[o.i.SERVER] ISPN080004: Connector SinglePort (default) listening on x.x.x.x:11222

Thanks
Regards
Ismael :)

**Comment by tristantarrant** [comment, marked as answer]
You are right: it's a bug. Launch it with 
```
server.sh -Dinfinispan.bind.port=11232
```
as a workaround

**Reply by iboumedien** [reply]
@tristantarrant Thanks, It's working Merci ;)

**Reply by tristantarrant** [reply]
https://github.com/infinispan/infinispan/pull/12634
